ui/src/main/picsureui/overrides/footer.js

```diff
@@ -1,9 +1,25 @@
-define(["handlebars"], function(HBS){
-	return {
-		/*
-		 * The render function for the footer can be overridden here.
-		 */
-		
-		render : undefined
-	};
-});
\ No newline at end of file
+define(["handlebars",
+        "text!overrides/footer.hbs",
+        "common/modal",
+        "common/session",
+        "common/pic-sure-dialog-view",
+        "middleware/middleware",
+        "common/redirect-modal"
+    ],
+    function (HBS, template, modal, session, dialog, Middleware, redirectModal) {
+        return {
+            /*
+             * The render function for the footer can be overridden here.
+             */
+            render: function () {
+                new Middleware();
+                let redirect = new redirectModal();
+
+                // Using .off() to prevent multiple event handlers from being attached
+                $(document).off('click', 'a[target="_blank"]').on('click', 'a[target="_blank"]', function (event) {
```

Comment:
could this potentially effect any other links outside the footer?

Reply:
That is the intended purpose actually. This ensures that all anchors tags in our application that have _blank first open a modal notifying the user they will be redirected. The footer was just a convent place to put the logic because it exists everywhere.